tensorflow/lite/tools/benchmark/benchmark_tflite_model.cc

```diff
@@ -378,7 +407,10 @@ std::vector<Flag> BenchmarkTfLiteModel::GetFlags() {
                        "are not used."),
       CreateFlag<int32_t>(
           "optimize_memory_for_large_tensors", &params_,
-          "Optimize memory usage for large tensors with sacrificing latency.")};
+          "Optimize memory usage for large tensors with sacrificing latency."),
+      CreateFlag<std::string>(
+          "save_outputs_in_file", &params_,
```

Comment:
How about "output_filepath" ?

Reply:
> 

Thanks for suggesting the simpler option. I've picked it up.